frontend/src/components/molecules/MatchTeam/MatchTeam.tsx

```diff
@@ -39,32 +39,39 @@ const useStyles = makeStyles((theme) => ({
     inline: {
         display: 'inline',
     },
+
+    waitingForTeam: {
+        display: 'inline',
+        color: 'red'
+    },
+
 }));
 
 export const MatchTeam = (props: MatchTeamProps) => {
     const classes = useStyles();
 
-    const noNumber = "NN";
+    const noInformation = "Brak danych";
+    const WaitingForTeam = <span className={classes.waitingForTeam}>"Oczekiwanie"</span>;
     const player1 = props.player1 || "player1";
     const player2 = props.player2 || "player2";
     const avatarSymbol = (typeof props.teamNumber === "string" ? props.teamNumber[0] : props.teamNumber)
-    const teamNumber = avatarSymbol || noNumber;
-    const currentPlayerLevel = (props.currentPlayerLevel || props.currentPlayerLevel === 0) ? props.currentPlayerLevel : noNumber;
+    const teamName = props.teamNumber;
+    const currentPlayerLevel = (props.currentPlayerLevel || props.currentPlayerLevel === 0) ? props.currentPlayerLevel : noInformation;
     const currentMatchNumber = (props.currentMatchNumber || props.currentMatchNumber === 0) ? props.currentPlayerLevel
-        : noNumber;
-    const teamNameText = "Team " + teamNumber;
-    const playersNameText = <span> {player1} <br/> & {player2} </span>
+        : noInformation;
+    const teamNameTitle = <span>Team: <br/> {teamName ? teamName?.toString().slice(0,15):WaitingForTeam }...</span>;
```

Comment:
Co tutaj to robi :O ? 
`teamName?.toString().slice(0,15)`?

Reply:
Tzn. po co ten slice?

Reply:
Nie wiem jak inaczej ograniczyć nazwę Teamu aby nie zajmowała kilku linii - aktualnie nazwa zespołu to ID zespołu!

Reply:
Zdaje mi sie, ze to CSSem. Ale to Ty jestes mistrzem :) 
Cos w stylu:
https://stackoverflow.com/questions/17868503/on-css-if-text-line-is-break-show-dots

Reply:
ok :)